test_opensearchpy/test_async/test_server/test_helpers/test_faceted_search.py

```diff
@@ -81,6 +85,11 @@ class CommitSearch(AsyncFacetedSearch):
 
 @pytest.fixture(scope="function")  # type: ignore
 def repo_search_cls(opensearch_version: Any) -> Any:
+    """
+    creates a mock AsyncFacetedSearch instance to search a git repo
+    :param opensearch_version:
+    :return:
```

Comment:
missing param description

Reply:
actually going to remove the docstring altogether. we're not going to require it for tests because tests need to be read to be useful and functions should be self-describing (and they usually are)